compiler/rustc_middle/src/hir/map/mod.rs

```diff
@@ -1133,8 +1131,6 @@ pub(super) fn crate_hash(tcx: TyCtxt<'_>, _: LocalCrate) -> Svh {
         }
         tcx.sess.opts.dep_tracking_hash(true).hash_stable(&mut hcx, &mut stable_hasher);
         tcx.stable_crate_id(LOCAL_CRATE).hash_stable(&mut hcx, &mut stable_hasher);
-        // Hash visibility information since it does not appear in HIR.
-        resolutions.visibilities.hash_stable(&mut hcx, &mut stable_hasher);
```

Comment:
This is now hashed into `tcx.hir_crate(()).opt_hir_hash`? Or how do we ensure that changing visibilities still affect the crate hash?

Reply:
I have restored the previous visibility hashing logic in a slightly simplified form.